units/stats_task_end.go

```diff
@@ -133,7 +133,8 @@ func (j *collectTaskEndDataJob) Run(ctx context.Context) {
 		"status":               j.task.ResultStatus(),
 		"task":                 j.task.DisplayName,
 		"task_id":              j.task.Id,
-		"total_wait_secs":      j.task.FinishTime.Sub(j.task.ActivatedTime).Seconds(),
+		"start_time":           j.task.StartTime,
```

Comment:
do we not currently use total_wait_secs for any tracking?

Reply:
There are a handful of instances I know of, but I want to move more towards calculating these times explicitly from scheduled_time and start time, or unblocked_time and start_time, as the situation demands. After this gets deployed I'm going to go through the dashboard that I know uses this and switch things out.